src/View/Helper/HtmlHelper.php

```diff
@@ -185,6 +185,7 @@ public function addCrumb($name, $link = null, array $options = [])
      * @param string $type Doctype to use.
      * @return string|null Doctype string
      * @link https://book.cakephp.org/3/en/views/helpers/html.html#creating-doctype-tags
+     * @deprecated 3.9.0 This method will be removed in 4.0.0.
```

Comment:
Should we also add a deprecation warning?

Reply:
I don't think that is necessary in those trivial cases.

Reply:
@dereuromark suggested just adding deprecation tag so that's what I did. Also didn't we decided to avoid throwing new warning in 3.9?

I don't care either way, happy to do whatever is preferred.

Reply:
This is usually more useful if there is a replacement to be done.
For just dropped stuff not sure how useful those are.

Reply:
The warnings let people avoid fatal errors before they upgrade.

Reply:
Fair enough

Reply:
I have added explicit warning.